src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -471,20 +472,23 @@ void QMCFixedSampleLinearOptimizeBatched::process(xmlNodePtr q)
   std::string vmcMove("pbyp");
   std::string ReportToH5("no");
   std::string OutputMatrices("no");
+  std::string OutputMatricesHDF("no");
   std::string FreezeParameters("no");
   OhmmsAttributeSet oAttrib;
   oAttrib.add(useGPU, "gpu");
   oAttrib.add(vmcMove, "move");
   oAttrib.add(ReportToH5, "hdf5");
 
   m_param.add(OutputMatrices, "output_matrices");
+  m_param.add(OutputMatricesHDF, "output_matrices_hdf");
```

Comment:
Please modernize all the "add" as
https://github.com/QMCPACK/qmcpack/blob/9dfd387e343ed0ecab8583a7de4c26baba225794/src/io/OhmmsData/AttributeSet.h#L58
Use the third argument to restrict input options.

Reply:
Fixed